livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -80,7 +80,8 @@ public InterpreterResult interpret(String line, InterpreterContext interpreterCo
               line.replaceAll("\"", "\\\\\"")
                   .replaceAll("\\n", " ")
               + "\").show(" +
-              property.get("zeppelin.livy.spark.sql.maxResult") + ")",
+              property.get("zeppelin.livy.spark.sql.maxResult") + "," +
+              property.get("zeppelin.livy.spark.sql.truncateResult") + ")",
           interpreterContext, userSessionMap);
```

Comment:
What kind of exception do you see in frontend ? If user set an invalid value like 'F' and if the exception in frontend doesn't imply that, then it might be better to verify the property in method `open()`, otherwise it is hard to figure out what is wrong.